includes/class-cache.php

```
@@ -119,7 +137,14 @@ public function __construct() {
 
 			$this->url_path = $url_path;
 
-			$this->options = get_option( 'wppo_settings', array() );
+			// Initialize filesystem and options.
+			$this->filesystem     = Util::init_filesystem();
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'PerformanceOptimise\Inc\Util' in method '__construct'. (undefined)

(StaticAccess)